packages/flutter/lib/fix_data.yaml

```diff
@@ -910,7 +910,7 @@ transforms:
             name: 'autovalidateMode'
             style: optional_named
             argumentValue:
-              expression: 'AutoValidateMode.disabled'
+              expression: 'AutovalidateMode.disabled'
               requiredIf: "autovalidate == 'false'"
           - kind: 'removeParameter'
             name: 'autovalidate'
```

Comment:
Below here, under `variables` you should be able to add the import if I understand it right
Like: 
```
kind: import
uris: [ 'package:flutter/material.dart' ]
name: 'Widget'
```

I haven't found an example of this yet, but I am looking into it now. Thanks @bwilkerson!

Reply:
I can add this in a follow-up change, it'll take a bit more refactoring I think. :)